include/dpp/user.h

```diff
@@ -312,6 +312,14 @@ class DPP_EXPORT user : public managed, public json_interface<user>  {
 	 * @return Formatted username and discriminator
 	 */
 	std::string format_username() const;
+	
+	/**
+	 * @brief Compare two user objects.
+	 * @param other_user Other user to compare to.
+	 * @return true if the user ids are equal, false otherwise.
+	 */
+	
+	bool operator == (user const& other_user) const;
```

Comment:
this is not needed. user derives from `managed` which already has `operator ==` and `operator !=` (see `dpp/managed.h`).

Reply:
Removed